pageserver/src/bin/pageserver.rs

```diff
@@ -87,8 +87,10 @@ fn main() -> anyhow::Result<()> {
     };
 
     // initialize sentry if SENTRY_DSN is provided
-    let _sentry_guard =
-        init_sentry(&[("process", "pageserver"), ("node_id", &conf.id.to_string())]);
+    let _sentry_guard = init_sentry(
+        release_name!(),
+        &[("process", "pageserver"), ("node_id", &conf.id.to_string())],
```

Comment:
Here and forth: why add `("process", <NAME>)` when `release_name!` already does that?

Reply:
Initially, I thought it was added to be able to group by captures by tag. But, we've decided to go with a sentry URL per "process". Therefore it's useless anymore. Will delete it.